### PR TITLE
Modified lesson3.exercise1, 10th of July

### PR DESCRIPTION
**Pull request by artpetrovskii**

### CODE REVIEW

File: lesson 3. exercise 1.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
-def game():
+    import random
```
Comment (apetrovskiy) [inline lesson 3. exercise 1.py]: It doesn't even work:
```  File "C:/Projects/Python/Basics-Python-Course/lesson 3. exercise 1.py", line 1
    import random
    ^
IndentationError: unexpected indent

**Review by apetrovskiy — CHANGES_REQUESTED** [review]

File: lesson 3. exercise 1.py [hunk 2]
```diff
@@ -1,48 +1,47 @@
-def game():
```
Comment (apetrovskiy) [inline lesson 3. exercise 1.py:1]: Now the program works differently, how will you merge the past and the current versions?

File: lesson 3. exercise 1.py [hunk 3]
```diff
@@ -1,48 +1,47 @@
...
+        print(fails[random.randint(1, 3)])
+        begin = version + 1
+        is_repeat = False
+    elif user_answer == '9':
+        print(fails[random.randint(1, 3)])
+        end = version - 1
+        is_repeat = False
+    else:
+        print(angry[random.randint(1, 3)])
+        is_repeat = True
```
Comment (apetrovskiy) [inline lesson 3. exercise 1.py:47]: It works the wrong way:
```Задумайте любое число от 1 до 100 и запишите его на бумаге. 
 Как будете готовы, нажмите 'ввод'
Ммм... может 62 загаданное число?
1: Ты угадал! 5: Недобор! 9: Перебор!
5
Ну как так
Я предположу, что вы загадали 89?
1: Ты угадал! 5: Недобор! 9: Перебор!
5
Черт
Ммм... может 100 загаданное число?
1: Ты угадал! 5: Недобор! 9: Перебор!
5
О нет
Ненавижу тебя!

**Review by apetrovskiy — CHANGES_REQUESTED** [review]

File: lesson 3. exercise 1.py (hunk 3 above)
Comment (artpetrovskii) [reply lesson 3. exercise 1.py:47]: I fixed this problem

File: lesson 3. exercise 1.py [hunk 4]
```diff
@@ -1,32 +1,46 @@
-def game():
-    print("Я программа угадывающая числа от 1 до 100")
-    input("Задумайте любое число от 1 до 100 и запишите его на бумаге. \n Как будете готовы, нажмите 'ввод'")
+import random
+
+    begin = 1
```
Comment (apetrovskiy) [inline lesson 3. exercise 1.py]: image

**Review by apetrovskiy — APPROVED** [review]

**Pull request merged**